docs/BUILD_AND_TEST.md

```diff
@@ -1,26 +1,54 @@
  
 # Build and Test
 ## Getting Started
+
 To build snap you'll need:
+
 * [Golang >= 1.4](https://golang.org)
```

Comment:
We don't support 1.4 any further. 1.5.3+ is where I believe we are. Could you confirm @geauxvirtual ?